tests/url_preview_test.go

```diff
@@ -0,0 +1,108 @@
+package tests
```

Comment:
Please remove this file and the test within. Complement does not listen on `:8448` anymore so this won't work, and I'd want to have a standardised way of spinning up mini web-servers.

Reply:
What do you mean with "test within"?

And i'll try to figure out what a nice standardized way could be, but iirc this was the best way to do it when i wrote this, but that could've changed by now.

Reply:
Basically remove the test and file and don't replace it with something else.

Reply:
Why exactly? Should "Test URL preview" not be tested? Or URL previews at all?

Reply:
They should be tested, but only once we have a standardised way to test with webservers like this.

Reply:
:+1: Alright, then i'll remove the tests related to URL previews, and make a note to add in that standardized way first, before i'll re-add them.